tests/testthat/test-indetion_round_brackets.R

```diff
@@ -0,0 +1,68 @@
+context("Function calls with round brackets")
+
+test_that("one-line function call yields correct indention", {
+  code <- "a <- xyz(x, 22, if(x > 1) 33 else 4)"
+
+  back_and_forth <- code %>%
+    compute_parse_data_nested() %>%
+    create_filler_nested() %>%
+    indent_round_nested() %>%
+    strip_eol_spaces_nested() %>%
+    serialize_parse_data_nested()
+
+  expect_identical(code, back_and_forth)
+})
+
+##  ............................................................................
+
+indented_multi_line_correct <- c(
```

Comment:
I wonder if this is easier in example files (like `in.R` and `out.R`). Perhaps you can extend to support an arbitrary number of `in.R` files? Again, separate PR.